CHANGELOG.md

```diff
@@ -5,6 +5,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [v7.7.1]
+
+### Fixed
```

Comment:
Isn't this now Added? 🫣

Reply:
I still see this as fixed since you could already set the double encoding to true even if it was an oAuth1 client. It was just broken.

Reply:
For me we added a new feature being a new method to sign OAuth1 requests AND we fixed OAuth1 signature exceptions when using double encoded characters

Just my two cents, I'll approve regardless.

Reply:
Now that we've agreed to go for a 7.8.0 version this could indeed be listed as `Changed`?